### PR TITLE
Feature/eslint

### PR DESCRIPTION
**Pull request by Amir-61**
@superkhau Please review :-)

### CODE REVIEW

**Comment by superkhau** [comment]
LGTM

File: index.js [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
+  var path = require('path');
+  var urlJoin = require('./lib/url-join');
+  var _defaults = require('lodash').defaults;
+  var cors = require('cors');
+  var createSwaggerObject = require('loopback-swagger').generateSwaggerSpec;
+  var SWAGGER_UI_ROOT = require('strong-swagger-ui/index').dist;
+  var STATIC_ROOT = path.join(__dirname, 'public');
+
+  module.exports = explorer;
+  explorer.routes = routes;
```
Comment (bajtos) [inline index.js]: @Amir-61 @superkhau this has re-indented whole `index.js` by one level of nesting :(

File: index.js [hunk 2]
```diff
@@ -21,70 +21,70 @@ explorer.routes = routes;
...
-  var router = new loopback.Router();
+    var router = new loopback.Router();
 
-  mountSwagger(loopbackApplication, router, options);
+    mountSwagger(loopbackApplication, router, options);
 
   // config.json is loaded by swagger-ui. The server should respond
   // with the relative URI of the resource doc.
-  router.get('/config.json', function(req, res) {
+    router.get('/config.json', function(req, res) {
```
Comment (bajtos) [inline index.js]: At the same time, many comments were not reindented, which creates a real mess.

File: index.js (hunk 1 above)
Comment (Amir-61) [inline index.js]: Oh that's weird! Sorry I did not notice that...

Comment (superkhau) [inline index.js]: @bajtos I didn't notice it. Not a big deal though, let's submit another PR to fix it. ;)
